website/pages/en/sunrise.mdx

```diff
@@ -1,233 +1,79 @@
 
-### Should I use a centralized indexing provider?
+### How does this affect subgraph developers?
 
-If you are building in web3, the moment you use a centralized indexing provider, you are giving them control of your dapp and data. The Graph’s decentralized network offers [superior quality of service](https://thegraph.com/blog/qos-the-graph-network/), reliability with unbeatable uptime thanks to node redundancy, significantly [lower costs](/network/benefits/), and keeps you from being hostage at the data layer.
+Subgraph developers can query their subgraphs on The Graph Network almost immediately after upgrading from the hosted service or [publishing from Subgraph Studio](https://thegraph.com/docs/en/publishing/publishing-a-subgraph/), as no lead time was required for indexing. Please note that [creating a subgraph](https://thegraph.com/docs/en/developing/creating-a-subgraph) was not impacted by this upgrade.
 
-With The Graph Network, your subgraph is public and anyone can query it openly, which increases the usage and network effects of your dapp.
+### How does the upgrade Indexer benefit data consumers?
 
-Additionally, Subgraph Studio provides 100,000 free monthly queries on the Free Plan, before payment is needed for additional usage.
+The upgraded Indexer enables chains on the network that were previously only supported on the hosted service. Therefore, it widens the scope and availability of data that can be queried on the network.
 
-Here's a detailed breakdown of the benefits of The Graph over centralized hosting:
+### How does the upgrade Indexer price queries?
 
-- **Resilience and Redundancy**: Decentralized systems are inherently more robust and resilient due to their distributed nature. Data isn't stored on a single server or location. Instead, it's served by hundreds of independent Indexers around the globe. This reduces the risk of data loss or service interruptions if one node fails, leading to exceptional uptime (99.99%).
+The upgrade Indexer prices queries at the market rate to avoid influencing the query fee market.
 
-- **Quality of Service**: In addition to the impressive uptime, The Graph Network features a ~106ms median query speed (latency), and higher query success rates compared to hosted alternatives. Read more in [this blog](https://thegraph.com/blog/qos-the-graph-network/).
+### When will the upgrade Indexer stop supporting a subgraph?
 
-- **Censorship Resistance**: Centralized systems are targets for censorship, either through regulatory pressures or network attacks. In contrast, the dispersed architecture of decentralized systems makes them much harder to censor, which ensures continuous data availability.
+The upgrade Indexer serves a subgraph until it is sufficiently and successfully served with consistent queries served by at least 3 other Indexers.
```

Comment:
Hmm, repeating the word "served" sounds like a potential mistake?

Reply:
Maybe this is what we mean?
```suggestion
The upgrade Indexer supports a subgraph until at least 3 other Indexers successfully and consistently serve queries made to it.
```

Reply:
would be fair to state "The upgrade Indexer serves a subgraph until it is sufficiently and successfully supported with consistent queries by at least 3 other Indexers"

Reply:
Hmm, not sure what "successfully supported" means. IMO, the "success" aspect refers to query success rate.

Reply:
I also feel like "supports a subgraph" is more correct than "serves a subgraph". An Indexer serves _queries_ made to a subgraph, not a subgraph itself, right?

Reply:
correct!